grid2op/Space/GridObjects.py

```diff
@@ -4198,5 +4262,9 @@ class {cls.__name__}({cls._INIT_GRID_CLS.__name__}):
     alarms_lines_area = {alarms_lines_area_str}
     alarms_area_lines = {alarms_area_lines_str}
 
+    # alert feature
+    dim_alert = {cls.dim_alerts}
+    alertable_line_names = {alertable_line_names_str}
```

Comment:
This whole function should be tested somewhere (maybe it is, I just put a remark here not to forget)